packages/Webkul/Checkout/src/Cart.php

```diff
@@ -1121,6 +1143,32 @@ private function fillAddressAttributes(array $addressAttributes): array
         return $attributes;
     }
 
+    /**
+     * Remove item from cart, whose product is inactive
+     * and returns true, if so.
+     *
+     * @param \Webkul\Checkout\Models\CartItem $item
+     *
+     * @return bool
+     */
+    private function removeInactiveItem(\Webkul\Checkout\Models\CartItem $item): bool
```

Comment:
As we talked about, I call removeInactiveItem() in isItemsHaveSufficientQuantity() too.
Combining of removeInactiveItem**s**() and removeInactiveItem() would cause an additional run through all cart items .

With this approach this is not necessary.

Reply:
Done